test/visualize/test_visualize.py

```diff
@@ -311,7 +312,9 @@ def test_profiles_with_options():
     pypesto.visualize.profiles(result,
                                reference=ref_point,
                                size=alt_fig_size,
-                               colors=[1., .3, .3, 0.5])
+                               profile_list_ids=[0, 1],
+                               legends=['profile list 0', 'profile list 1'],
+                               colors=[[1.,.3,.3,.5], [.5,.9,.4,.3]])
```

Comment:
we have an issue somewhere that we want more flexible colors, but fine for the moment ...